src/main/java/edu/harvard/dbmi/avillach/dictionary/dashboarddrawer/DashboardDrawerRepository.java

```diff
@@ -14,19 +12,14 @@ public class DashboardDrawerRepository {
 
     private final NamedParameterJdbcTemplate template;
 
-    private static final Logger log = LoggerFactory.getLogger(DashboardDrawerRepository.class);
-
     @Autowired
     public DashboardDrawerRepository(NamedParameterJdbcTemplate template) {
         this.template = template;
     }
 
-    public List<DashboardDrawer> getDashboardDrawerRows() {
-        String materializedViewSql = """
-            select * from dictionary_db.dict.dataset_meta_materialized_view dmmv;
-            """;
+    public Optional<List<DashboardDrawer>> getDashboardDrawerRows() {
```

Comment:
Debug log is too chatty. removing materialized views as they are not in use anyways.